From 950867d9fd3f690e271c8c807b6eed144b2935b2 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 27 Aug 2018 15:00:51 +0800
Subject: [PATCH] configure.in: remove useless `-L$use_openssl/lib'

Since `--with-openssl=${STAGING_DIR_HOST}${prefix}' is used in bind recipe,
the `-L$use_openssl/lib' has a hardcoded suffix, removing it is harmless
and helpful for clean up host build path in isc-config.sh

Upstream-Status: Inappropriate [oe-core specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 54efc55..76ac0eb 100644
--- a/configure.in
+++ b/configure.in
@@ -1691,7 +1691,7 @@ If you don't want OpenSSL, use --without-openssl])
 				fi
 				;;
 			*)
-				DST_OPENSSL_LIBS="-L$use_openssl/lib -lcrypto"
+				DST_OPENSSL_LIBS="-lcrypto"
 				;;
 			esac
 		fi
-- 
2.7.4

